package.json

```diff
@@ -4,9 +4,9 @@
   "description": "A simple typescript jsfeatNextt example.",
   "main": "index.js",
   "scripts": {
-    "dev-ts": "rimraf ./build && tsc && webpack --mode development --progress --watch",
```

Comment:
this was wrong rimraf should delete the folder in the command but since we have no build folder will not delete anything...